filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+# FileWriter V2
+
+---
+$\color{red}\text{\large{NOTE: FileWriter V2 is currently in a preview state and will change in the future.}}$
+
+Feedback is very welcome!
+
+---
+
+FileWriter writes image data and metadata to [HDF5](https://www.hdfgroup.org/solutions/hdf5/) files. Currently, there are two different formats available. V1 is a legacy format that is loosely based on [version 1.4](https://github.com/nexusformat/definitions/releases/tag/NXmx-1.4) of the _NXmx application definition_ of the [NeXus] format. V2 follows the [_NXmx application definition_](https://manual.nexusformat.org/classes/applications/NXmx.html) of NeXus Version 2022.07, with the addition of multi-channel data support[^1].
+
+| Version | Configuration | NeXus/NXmx Version | Multi-channel data support |
+|----|----|----|----|
+| V1 | SIMPLON | loosely based on [NXmx 1.4](https://github.com/nexusformat/definitions/releases/tag/NXmx-1.4) | no |
+| V2 | SIMPLON | [NeXus 2022.07 NXmx](https://github.com/nexusformat/definitions/blob/v2022.07/applications/NXmx.nxdl.xml) | yes |
+
+[^1]: This is not part of the NeXus standard yet, a detailled PR will be opened when this data format has been approved by the community. The gerenal idea of this representation has already been discussed and approved by the NIAC though, see this [issue](https://github.com/nexusformat/definitions/issues/940).
```

Comment:
type: "general"

Reply:
fixed in [34890fd](https://github.com/soph-dec/documentation/pull/1/commits/34890fdc03430a5b054a735918d0593e38a8e157)